docs/cylc-8-roadmap.md

```diff
@@ -13,7 +13,23 @@ meetings, but not in a way that will hurt you._
 We need something very much like the [Jupyter
 Hub](https://jupyterhub.readthedocs.io/en/stable/) Architecture: a privileged
 hub that spawns a reverse proxy server and sub-services for suite discovery etc.,
-and spawns suite daemons (c.f. Jupyter notebooks) as the user.
+and spawns UI servers (c.f. Jupyter notebooks) and potentially suite daemons as
+the user.
+
+### Core Principles & Motivation For The New Architecture
```

Comment:
The first section under 'Architectural Challenges' in the ``gui-replacement-options.md`` file discusses this topic - can we move one of these two sections/lists so they are in the same place, or even collate them together, to keep the material organised? @hjoliver I personally feel the 'Architectural Challenges' would be better placed in this file, as a preference for movement direction.

Reply:
@sadielbartholomew - sounds sensible.  I'll merge this PR and then tweak along the lines you suggest.